aten/src/ATen/core/boxing/impl/make_boxed_from_unboxed_functor.h

```diff
@@ -59,13 +59,8 @@ namespace impl {
       guts::if_constexpr<guts::typelist::contains<supported_primitive_arg_types, T>::value>([] {
         /* everything is ok, this is a primitive type */
       }, /* else */ [] {
-        // TODO This is called for each operator call and potentially expensive.
```

Comment:
We could still have a `std::is_base_of` check here just to make sure it actually inherits from the custom base class.

Reply:
Hmm looks like `is_base_of` reveals some use of classes that aren't in fact subclasses of `CustomClassHolder`. Since we're not actually reducing the test coverage if we omit the static tests from this PR, I'd rather land it as-is for the deduplication, and put the static checks (and whatever fixes are required) in a followup. Will add that one to the stack with additional info.